GG/src/RichText/BlockControl.cpp

```diff
@@ -0,0 +1,32 @@
+/** \file \brief The implementation of BlockControl.
+ */
+
+#include <GG/RichText/BlockControl.h>
+
+namespace GG {
+
+    BlockControl::BlockControl(X x, Y y, X w, GG::Flags<GG::WndFlag> flags)
+    : Control(x, y, w, Y0, flags)
```

Comment:
: to end of previous line, indent Control by 4 spaces, put { and } on the same line.